### ISSUE TITLE
api_debug causes Frozen string error

### ISSUE THREAD

**Issue by apuntovanini** [jarthod/survey-gizmo-ruby#111]
When we run survey-gizmo-ruby with debug everything crashes with "Can't modify frozen string".
Error comes from https://github.com/jarthod/survey-gizmo-ruby/blob/master/lib/survey_gizmo/logger.rb#L7
It's probably sufficient to remove the hashbang from gsub
What do you think?

**Comment by jarthod** [comment]
Indeed, could you please write a PR to change this to a normal gsub? it's not a good practice to modify arguments anyway.
